tests/Unit/Utilities/PregTest.php

```diff
@@ -0,0 +1,219 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Pelago\Emogrifier\Tests\Unit\Utilities;
+
+use Pelago\Emogrifier\Utilities\Preg;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * @covers \Pelago\Emogrifier\Utilities\Preg
+ */
+final class PregTest extends TestCase
+{
+    /**
+     * @test
+     */
+    public function throwExceptionsProvidesFluentInterface(): void
+    {
+        $subject = new Preg();
+
+        $result = $subject->throwExceptions(false);
+
+        self::assertSame($subject, $result);
+    }
+
+    /**
+     * @test
+     */
+    public function exceptionNotThrownByDefault(): void
```

Comment:
Let's make the test name into a full statement:

```suggestion
    public function replaceByDefaultDoesNotThrowAnException(): void
```

Reply:
I've not renamed this to explicitly state the method that will be tested for not throwing an exception.  I realized that these tests need to cover that the exceptions on/off switch applies to both methods.  So have extended the tests thusly.